velox/common/file/tests/FileTest.cpp

```diff
@@ -115,6 +115,27 @@ TEST(LocalFile, viaRegistry) {
   lfs->remove(filename);
 }
 
+TEST(LocalFile, rename) {
+  filesystems::registerLocalFileSystem();
+  auto tempFolder = ::exec::test::TempDirectoryPath::create();
+  auto a = fmt::format("{}/a", tempFolder->path);
```

Comment:
nit:
```
s/a/fileAPath/
s/b/fileBPath/
s/newA/newFileAPath/
```

Reply:
But there are multiple other unittests in this file that are organize that way,

Reply:
In terms of naming the files a or b